serverless.webpack.config.js

```diff
@@ -46,6 +46,10 @@ const ServerlessWebpackConfig = {
   ],
   module: {
     rules: [
+      {
+        test: /\.svg$/,
```

Comment:
Needed so I could read in an svg into serverless to return unavailable images during failures